### DISCUSSION TITLE
Lightbox/popup functionality

### DISCUSSION THREAD

**Discussion by puresystems** [vanjarosoftware/Vanjaro.Platform#2019] [Q&A] [answered]
Hi, is there a way to easily create muliple popups/lightboxes on a Vanjaro page through the UI?

**Comment by meetmandeep** [comment, marked as answer]
Not at the moment. That's on the roadmap as a popup builder app.

**Reply by puresystems** [reply]
OK great, thanks.

**Reply by puresystems** [reply]
Just looking at your ToolTip module - it pretty much does what the customer wants and easy to manage - could you look at adding in a lightbox effect to this?

**Reply by meetmandeep** [reply]
Which ToolTip module are you referring to? Vanjaro has nothing to do with existing products on Mandeeps.com

**Reply by puresystems** [reply]
ok, my mistake.
